### PR TITLE
REL-4166: Changed LTTS font sizes and box sizes to make status panel text fit.

### PR DESCRIPTION
**Pull request by sraaphorst**
We have long had a UI issue in the LTTS where the font size in the status panels has been too big, sometimes cutting off important parts of the data. Here is a normal status which fits regularly but is bordering on not fitting for the Beam Collision status:
ltts_gem

Here is an example where the text does not fit and information is cut off:
subarultcs

I have modified the CSS in order to shrink the text size down, and then tested with the largest possible value (I believe) to make sure that the status text does indeed fit. Here is the result of the test:
Screen Shot 2022-08-11 at 9 59 23 AM

As is visible, the Beam Collision status now can list Subaru with the maximum digit width.

This addresses the long-standing fault:
[https://osc.hi.gemini.edu/browse/FR-72358](https://osc.hi.gemini.edu/browse/FR-71795)

### CODE REVIEW

File: web/src/main/java/edu/gemini/lch/web/app/windows/alarm/StatusPanel.java [hunk 1]
```diff
@@ -237,7 +237,7 @@ public BeamCollision() {
                     // get first collision (sorted by time) and check if we need to react
                     // we need to react if a) laser is on sky and b) other telescope has priority
                     LtcsService.Collision c = collisions.get(0);
-                    Boolean needToReact = snapshot.getEpicsSnapshot().isOnSky() && !c.geminiHasPriority();
+                    boolean needToReact = snapshot.getEpicsSnapshot().isOnSky() && !c.geminiHasPriority();
```
Comment (cquiroz) [inline web/src/main/java/edu/gemini/lch/web/app/windows/alarm/StatusPanel.java:240]: Is this related to the fix?

Comment (sraaphorst) [reply web/src/main/java/edu/gemini/lch/web/app/windows/alarm/StatusPanel.java:240]: No... it was just irritating me like nails on a chalkboard. I can reverse it if you want.

Comment (cquiroz) [reply web/src/main/java/edu/gemini/lch/web/app/windows/alarm/StatusPanel.java:240]: don't worry I was just curious

**Review by cquiroz — APPROVED** [review]
LGTM
